components/wpt-results.html

```diff
@@ -46,59 +120,170 @@
 
       static get properties() {
         return {
+          query: {
+            type: String,
+            value: '',
+            observer: '_queryChanged'
+          },
           specs: {
             type: Array,
-            observer: '_specsChanged',
             value: []
-          }
+          },
+          testRuns: {
+            type: Array,
+          },
         };
       }
 
-      connectedCallback() {
+      async connectedCallback() {
         super.connectedCallback();
 
-        this._fetchResults()
+        const testFileResults = await Promise.all(this.testRuns.map(testRun => {
+          return this._fetchResults(testRun.results_url)
+        }))
+
+        let specs = []
+
+        testFileResults.forEach(result => {
+          const testFiles = result.testFiles
+          const resultsURL = result.resultsURL
```

Comment:
I haven't run eslint or golint (or gofmt?) on this repo - I'll make a bug to add those.

Speaking of which - for bugs like adding eslint, are you ok with keeping them in this repo?